File: 2.0/java/calls/findCallBroadcasts.java
import com.callfire.api.client.CallfireClient;
import com.callfire.api.client.api.campaigns.model.CallBroadcast;
import com.callfire.api.client.api.campaigns.model.request.FindCallBroadcastsRequest;
import com.callfire.api.client.api.common.model.Page;

class ApiClientSample {
    public static void main(String[] args) {
        CallfireClient client = new CallfireClient("api_login", "api_password");
        FindCallBroadcastsRequest request = FindCallBroadcastsRequest.create()
            // find all call broadcasts which name like 'campaign'
            .name("broadcast name")
            .label("my label")
            .running(true)
            .offset(0L)
            .limit(2L)
            .fields("items(id,name,status,sounds)")
            .build();
        Page<CallBroadcast> broadcasts = client.callBroadcastsApi().find(request);
    }
}

File: 2.0/csharp/calls/findCallBroadcasts.cs
using CallfireApiClient;
using CallfireApiClient.Api.Campaigns.Model.Request;

public class ApiClientSample
{
    public static void Main(string[] args)
    {
        var client = new CallfireClient("api_login", "api_password");

        var findRequest = new FindBroadcastsRequest
        {
            Name = "broadcast name",
            Label = "my label",
            Running = true,
            Offset = 0,
            Limit = 10,
            Fields = "items(id,name,status,sounds)"
        };
        var broadcasts = client.CallBroadcastsApi.Find(findRequest);
    }
}
